frontends/ol-util/assets/card.scss

```diff
@@ -0,0 +1,44 @@
+@use "variables";
```

Comment:
`@use` is the Sass module system version of `@important`, see https://sass-lang.com/blog/the-module-system-is-launched for its announcement.

Whereas `@import` imports stuff into the global scope, `@use` provides variables onto a namespace (e.g., `variables.$card-background` below).

Sort of related... if you `@import` the same thing many times, it gets included in the output css multiple time. With `@use`, it just gets included once. (Which make sense, since you'd need to `@use` the variables thing a bunch more). 

Apparently `@import` is being deprecated, but not super soon. https://github.com/sass/sass/blob/main/accepted/module-system.md#timeline

One big difference is that `@use` declarations have to be at the top of a sass file.